include/lbann/layers/transform/pooling.hpp

```diff
@@ -162,20 +160,14 @@ class pooling_layer : public transform_layer<TensorDataType> {
     m_strides = other.m_strides;
     m_max_pool_indices = other.m_max_pool_indices;
 #ifdef LBANN_HAS_CUDNN
-    copy_pooling_cudnn_desc(other.m_pooling_cudnn_desc, m_pooling_cudnn_desc);
```

Comment:
Since you remove the call to `copy_pooling_cudnn_desc`, then the function can likely be removed too?

Reply:
(Lines 543-589)